src/components/HelloWorld.vue

```diff
@@ -9,6 +9,10 @@ export default {
   name: 'HelloWorld',
   props: {
     msg: String
+  },
+  data () {
+    return {
+    }
```

Comment:
I think this is leftover from when you were showing us around earlier. Won't hurt anything to leave it here though, since we're going to be deleting this component at some point anyway.